.travis.yml

```diff
@@ -12,10 +12,12 @@ scala: 2.11.7
 
 jdk:
 - oraclejdk8
-- openjdk8
 
 script: 
-  - sbt ++$TRAVIS_SCALA_VERSION clean assembly 
+  - sbt ++$TRAVIS_SCALA_VERSION clean coverage test assembly
   
   # Trick to avoid unnecessary cache updates
   - find $HOME/.sbt -name "*.lock" | xargs rm
+
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
How does [codecov-bash](https://github.com/codecov/codecov-bash) differ `codecov` task described in their [Integration Guide](https://codecov.io/guide)?  Just to get around the `pip install codecov`, or different functionality?